app/admin/cdr/cdr_exports.rb

```diff
@@ -282,11 +282,14 @@ def build_new_resource
                input_html: {
                  class: 'chosen-ajax',
                  data: {
-                   path: '/customers_auths/search_with_return_external_id?q[ordered_by]=name'
+                   path: '/customers_auths/search_with_return_external_id?q[ordered_by]=name&q[external_id_null]=false'
                  }
                },
-               collection: if params.dig(:q, :customer_auth_external_id_in)
-                             CustomersAuth.where(external_id: params.dig(:q, :customer_auth_external_id_in)).order(:name).pluck(:name, :external_id)
+               collection: if params.dig(:cdr_export, :filters, :customer_auth_external_id_in)
```

Comment:
instead of params checking try to get values like this `ff.object.customer_auth_external_id_in`